packages/backend/src/apps/pair/index.ts

```diff
@@ -0,0 +1,16 @@
+import { IApp } from '@plumber/types'
+
+import actions from './actions'
+
+const app: IApp = {
+  name: 'Pair',
+  key: 'pair',
+  baseUrl: 'https://pair.gov.sg',
+  apiBaseUrl: 'https://staging.pair.gov.sg/api/v1',
```

Comment:
should this URL be pointing to staging?

Reply:
Yup right now the Pair API is not in prod yet. We will deploy this version to staging only, and make sure Pair got prod setup before our production release